pkg/iac-providers/kubernetes/v1/normalize.go

```diff
@@ -159,47 +159,16 @@ func readSkipRulesFromAnnotations(annotations map[string]interface{}, resourceID
 		return nil
 	}
 
-	skipRules := make([]output.SkipRule, 0)
-	if rules, ok := skipRulesFromAnnotations.([]interface{}); ok {
-		for _, rule := range rules {
-			if value, ok := rule.(map[string]interface{}); ok {
-				skipRule := getSkipRuleObject(value)
-				if skipRule != nil {
-					skipRules = append(skipRules, *skipRule)
-				}
-			} else {
-				zap.S().Debugf("each rule in %s must be of map type", terrascanSkip)
-			}
-		}
-	} else {
-		zap.S().Debugf("%s must be an array of {rule: ruleID, comment: reason for skipping}", terrascanSkip)
-	}
-
-	return skipRules
-}
-
-func getSkipRuleObject(m map[string]interface{}) *output.SkipRule {
-	var skipRule output.SkipRule
-	var rule, comment interface{}
-	var ok bool
-
-	// get rule, if rule not found return nil
-	if rule, ok = m[terrascanSkipRule]; ok {
-		if _, ok = rule.(string); ok {
-			skipRule.Rule = rule.(string)
-		} else {
+	if rules, ok := skipRulesFromAnnotations.(string); ok {
+		skipRules := make([]output.SkipRule, 0)
+		err := json.Unmarshal([]byte(rules), &skipRules)
+		if err != nil {
+			zap.S().Debugf("terrascanSkip rules '%s' cannot be unmarshalled to json", rules)
```

Comment:
```suggestion
			zap.S().Debugf("json string %s cannot be unmarshalled to [ ]output.SkipRules struct schema", rules)
```

Reply:
Devang's comment is technically accurate, but not sure if that much detail would be useful to average user, or confusing?

Reply:
I agree with @jlk. This change will not be informative to most users.

Reply:
Debug is also used by us developers.. to diagnose what went wrong. Will ease it for us to help people reporting bugs and other issues.